MathGame.JsPeanut/MathGame.JsPeanut/Program.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+
+class Program
+{
+
+    public static void Main(string[] args)
+    {
+        List<Operation> previousGames = new List<Operation>();
```

Comment:
We can shorten this to the following:
```suggestion
        List<Operation> previousGames = new();
```